### PR TITLE
Feat: enable base sepolia

### LINKED ISSUE #3144: Network support of Base Sepolia
## What is the feature about

Base Testnet Sepolia is being added to the Safe deployments (https://github.com/safe-global/safe-deployments/pull/362) and should be supported by the interface. 

Timeline tbd 

## The list of requirements

- test safe creation and tx execution @liliya-soroka, testing will be carried out by @JagoFigueroa  
- add Base Sepolia to the network list
- activate Base Sepolia on Prod 9th of Feburary

### PR DESCRIPTION
**Pull request by schmanu**
## What it solves

Resolves #3144 

## How this PR fixes it
- Adds icon to Base Sepolia
- Updates safe-core-sdk packages and safe-deployments

## How to test it
- Create a Safe on Sepolia

## Checklist
* [x] I've tested the branch on mobile 📱
* [x] I've documented how it affects the analytics (if at all) 📊
* [x] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Comment by liliya-soroka** [comment]
@schmanu ,in addition to the usual checks for a new chain I have checked:
1. signing in 2 of N and 3 of N safes
2. signing and execution with Ledger 
3.  messages signing using eip-1271-test-dapp ( 2 of N safe)

File: src/services/tx/tx-sender/create.ts [hunk 1]
```diff
@@ -79,7 +79,13 @@ export const createExistingTx = async (
   // Create a tx and add pre-approved signatures
   const safeTx = await createTx(txParams, txParams.nonce)
   Object.entries(signatures).forEach(([signer, data]) => {
-    safeTx.addSignature({ signer, data, staticPart: () => data, dynamicPart: () => '' })
+    safeTx.addSignature({
+      signer,
+      data,
+      staticPart: () => data,
+      dynamicPart: () => '',
+      isContractSignature: false,
```
Comment (usame-algan) [inline src/services/tx/tx-sender/create.ts:87]: What does this do? I can't find a reference in the protocol-kit repo

Comment (schmanu) [reply src/services/tx/tx-sender/create.ts:87]: The new alpha version supports EIP-1271. So a signature can be a contract signature (EIP-1271 Signature).
In that case the dynamic part would be filled etc.

I did not add 1271 support yet in this PR as this focused on Base Sepolia. That's why it is always false and we still sign on-chain with SmartContractWallets.

**Review by usame-algan — APPROVED** [review]
As discussed lets update to the stable version 🪨 🚀

**Pull request merged**